src/stories/Library/Modals/modal-profile/ModalProfile.stories.tsx

```diff
@@ -0,0 +1,66 @@
+import { withDesign } from "storybook-addon-designs";
+import { ComponentMeta, ComponentStory } from "@storybook/react";
+
+import { ModalProfile } from "./ModalProfile";
+
+export default {
+  title: "Library / Modals / Profile",
+  component: ModalProfile,
+  decorators: [withDesign],
+  argTypes: {},
+  parameters: {
+    design: {
+      type: "figma",
+      url: "https://www.figma.com/file/Zx9GrkFA3l4ISvyZD2q0Qi/Designsystem?node-id=2137%3A11410",
+    },
+  },
+} as ComponentMeta<typeof ModalProfile>;
+
+const Template: ComponentStory<typeof ModalProfile> = (args) => (
+  <ModalProfile {...args} />
+);
+
+export const Item = Template.bind({});
+Item.args = {
```

Comment:
Ever considered using argTypes with a default value set? (line 10) Then it's also added as a storybook control.
E.g. in the SearchResultItem.stories.tsx
```code
argTypes: {
    heartFill: {
      name: "Heart Fill", // name can also be added here for Storybook visualisation
      control: "boolean",
      defaultValue: false,
    },
```
(https://storybook.js.org/docs/react/api/argtypes)

Reply:
Thanks. That is absolutely the right way to do it. I have updated the story.